package.json

```diff
@@ -12,15 +12,32 @@
     "url": "https://github.com/eventualbuddha/keysim.js"
   },
   "scripts": {
-    "build": "gobble build dist -f",
-    "test": "npm run test:node && npm run test:browser",
-    "test:browser": "./node_modules/karma/bin/karma start karma.conf.js",
-    "test:node": "mocha -R spec --recursive test"
+    "build": "browserify . --standalone keysim > dist/keysim.js",
+    "test": "npm run build && npm run test:node && npm run test:node:dist && npm run test:browser",
+    "test:browser": "karma start --single-run",
+    "test:node": "mocha --require babel-register -R spec --recursive test",
+    "test:node:dist": "DIST=true mocha --require babel-register -R spec --recursive test"
   },
   "files": [
     "dist/keysim.js",
     "lib/keysim.js"
   ],
+  "browserify": {
+    "transform": [
+      "babelify"
+    ],
+    "ignore": [
+      "jsdom"
+    ]
+  },
+  "babel": {
+    "presets": [
+      "es2015"
+    ],
+    "plugins": [
+      "transform-class-properties"
```

Comment:
@eventualbuddha you were using static class properties (that are not methods) so here it is.